datahub-web-react/src/app/ingest/source/builder/RecipeForm/looker.ts

```diff
@@ -0,0 +1,29 @@
+import { RecipeField, FieldType } from './common';
+
+export const LOOKER_BASE_URL: RecipeField = {
+    name: 'base_url',
+    label: 'Base URL',
+    tooltip:
+        'Url to your Looker instance: https://company.looker.com:19999 or https://looker.company.com, or similar.Used for making API calls to Looker and constructing clickable dashboard and chart urls.',
```

Comment:
Add a space after the first sentence before "Used"

Reply:
Done!